hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/test/java/org/apache/hadoop/yarn/server/nodemanager/recovery/TestNMLeveldbStateStoreService.java

```diff
@@ -1756,6 +1756,19 @@ public void testStateStoreForResourceMapping() throws IOException {
     resources = rcs.getResourceMappings().getAssignedResources("numa");
     Assert.assertEquals(numaRes, resources);
     Assert.assertEquals(numaRes, resourceMappings.getAssignedResources("numa"));
+    // test removing numa resources from state store
+    stateStore.releaseAssignedResources(containerId, "numa");
+    restartStateStore();
```

Comment:
Why restartStateStore is needed here.

Reply:
Apologies ! this was added redundant ! removed it as this is not required